blivet/devicelibs/dasd.py

```diff
@@ -144,6 +144,30 @@ def is_ldl_dasd(device):
         return False
     return True
 
+def is_fba_dasd(device):
+    """Is the device a FBA DASD?"""
+    if not device.startswith("dasd"):
+        # not a dasd; bail
+        return False
+
+    device = "/dev/%s" % (device,)
```

Comment:
Fixing it on `2.1-devel` while minimizing changes for `rhel7-branch` sounds wise. (Also, you are correct about using `device.path`.)